tst/EnergyPlus/unit/HVACUnitarySystem.unit.cc

```diff
@@ -1710,35 +1715,35 @@ TEST_F(ZoneUnitarySystemTest, UnitarySystem_MultispeedPerformance)
     // autosized air flow and capacity, unitary sytsem capacity matches coils
     EXPECT_EQ(UnitarySystem(1).MaxCoolAirVolFlow, 1.5);
     EXPECT_EQ(UnitarySystem(1).MaxHeatAirVolFlow, 1.5);
-    EXPECT_NEAR(UnitarySystem(1).DesignCoolingCapacity, 31732.204, 0.001);
```

Comment:
Fan heat adjusted sizing and required any unit tests that test capacity to be revised.